core/flyout.js

```diff
@@ -427,9 +581,14 @@ Blockly.Flyout.prototype.show = function(xmlList) {
   this.listeners_.push(Blockly.bindEvent_(this.svgBackground_, 'mouseover',
       this, deselectAll));
 
-  this.width_ = 0;
+  if (this.horizontalLayout_) {
```

Comment:
I realize this just mirrors the width_ = 0 that was already there, but do you know why we set the width to zero when the mouse wheel is scrolling?